lib/pd.js

```diff
@@ -809,6 +961,44 @@ function Client(apiToken, tokenType, options) {
         }
     };
 
+// Service Dependencies
+// https://developer.pagerduty.com/api-reference/reference/REST/openapiv3.json/paths/~1service_dependencies~1associate/post
+this.serviceDependencies = {
+    associateDependencies: async(payload) => {
+        try {
+            return await request(this.options('/service_dependencies/associate', 'POST', null, payload))
+        }
+        catch (err) {
+            throw new Error(err)
+        }  
+    },    
+    disassociateDependencies: async(payload) => {
+        try {
+            return await request(this.options('/service_dependencies/disassociate', 'POST', null, payload))
+        }
+        catch (err) {
+            throw new Error(err)
+        }  
+    },    
+    getBusinessServiceDependencies: async(id) => {
+        try {
+            return await request(this.options('/service_dependencies/business_services/'+ id, 'POST', null, null))
+        }
+        catch (err) {
+            throw new Error(err)
+        }  
+    },    
+    getTechnicalServiceDependencies: async(id) => {
+        try {
+            return await request(this.options('/service_dependencies/technical_services/'+ id, 'POST', null, null))
+        }
+        catch (err) {
+            throw new Error(err)
+        }  
+    }
+};
```

Comment:
Can you also remove the leading / from options section here too `/service_dependencies/technical_services/`